cirq-google/cirq_google/serialization/circuit_serializer.py

```diff
@@ -670,7 +696,31 @@ def _deserialize_gate_op(
                 raise ValueError(f"dimensions {dimensions} for ResetChannel must be an integer!")
             op = cirq.ResetChannel(dimension=dimensions)(*qubits)
         elif which_gate_type == 'internalgate':
-            op = arg_func_langs.internal_gate_from_proto(operation_proto.internalgate)(*qubits)
+            parsed_as_stimcirq = False
```

Comment:
Can we move this code to a local `_deserialize_with_stimcirq_if_installed` function which would return `Optional[cirq.Operation]` - and if `None` it will fallback to `internal_gate_from_proto` below?

Reply:
After factoring out the import, this code block is pretty short.  No one gates have a local function.
Let me know if you still think this is a good idea.